addons/hr/models/hr_employee.py

```diff
@@ -305,19 +305,19 @@ def create(self, vals):
         return employee
 
     def write(self, vals):
-        if 'address_home_id' in vals:
-            account_id = vals.get('bank_account_id') or self.bank_account_id.id
-            if account_id:
-                self.env['res.partner.bank'].browse(account_id).partner_id = vals['address_home_id']
-            self.message_unsubscribe(self.address_home_id.ids)
-            if vals['address_home_id']:
+        for rec in self:
```

Comment:
Hey @niyasraphy :wave: 

Thanks for your fix ^^

Iterating through the recordset is really not a good idea in the write method. 
My colleague proposed another solution for it: https://gist.github.com/kba-odoo/4eba30d56348603f271849cf4fb0522e#file-diff-L25

What do you think? :smile:

Reply:
thanks @sofiagvaladze ,
indeed it was the easiest way, updated the PR